server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -167,20 +167,33 @@ interface IMongoDBConfig {
     bufferMaxEntries: number | undefined;
     globalDbEndpoint?: string;
     globalDbEnabled?: boolean;
+    connectionPoolMinSize?: number;
+    connectionPoolMaxSize?: number;
 }
 
 export class MongoDbFactory implements core.IDbFactory {
     private readonly operationsDbEndpoint: string;
     private readonly bufferMaxEntries?: number;
     private readonly globalDbEndpoint?: string;
+    private readonly connectionPoolMinSize?: number;
+    private readonly connectionPoolMaxSize?: number;
     constructor(config: IMongoDBConfig) {
-        const { operationsDbEndpoint, bufferMaxEntries, globalDbEnabled, globalDbEndpoint } = config;
+        const {
+            operationsDbEndpoint,
+            bufferMaxEntries,
+            globalDbEnabled,
+            globalDbEndpoint,
+            connectionPoolMinSize,
```

Comment:
Since we are adding `connectionPoolMinSize` and `connectionPoolMaxSize`, do we also need to update whoever calls this constructor to provide the new parameters? Or do we only want to do that in FRS?

Reply:
The parameters are directly load from config json and use object deconstruction to assign to variables. So all I need is to add these setting to `config.json` files for local and configmaps for cluster